addons/mail/static/src/core/common/composer.js

```diff
@@ -209,16 +209,18 @@ export class Composer extends Component {
     }
 
     get CANCEL_OR_SAVE_EDIT_TEXT() {
+        const str = this.ui.isSmall
+            ? "<button class='%(button_class)s' data-type='%(cancel_type)s'><i class='%(cancel_icon)s'></i><em>Discard editing</em></button>"
+            : "<samp>%(cancel_keybind)s</samp><i> to <a href='#' data-type='%(cancel_type)s'>cancel</a></i>, <samp>%(save_keybind)s</samp><i> to <a href='#' data-type='%(save_type)s'>save</a></i>";
```

Comment:
This broke translation, it can't be a variable, and it can't have a condition inside it. Only direct string inside `_t`.

Also this way of doing it (already before diff) is not safe in regards to markup.
What is received from the translation should always be escaped. HTML tags should not be exported for translation, what should be exported instead are placeholder for tags that we replace afterwards.